docs/architecture/smart-rollups.md

```diff
@@ -1,1310 +1,147 @@
-Status: Evaluating
-Internal state: Snapshot
-```
+At the end of the commitment period, the rollup node must use the PVM to publish its state to layer 1, which is called its _commitment_.
+Each commitment builds on the previous commitment, going back to the genesis commitment from when the Smart Rollup was originated.
+This commitment asserts that the rollup node has responded to every message in every block in the commitment period and updated its state according to the code for the Smart Rollup.
 
-Once again, note that values from the memory are output as is, since the representation is internal to WASM.
+Each node must stake 10,000 tez along with its commitment.
+When another node makes an identical commitment, its stake is added to the stake of the first commitment.
 
-Finally, it is possible to evaluate the whole inbox with `step inbox`.
-It will take care of the possible reboots asked by the kernel (through
-the usage of the `/kernel/env/reboot_flag` flag) and stop at the next
-collection phase:
+Then a refutation period for the commitment period begins, and a new commitment period begins.
 
-```sh
-> step inbox
-Evaluation took 44000000000 ticks
-Status: Waiting for inputs
-Internal state: Collect
-```
+### Refutation periods
 
-It is also possible to show the outbox for any given level:
+Because the PVM is deterministic and all of the inputs are the same for all nodes, any honest node that runs the same Smart Rollup produces the same commitment.
+As long as nodes publish matching commitments, they continue running normally.
 
-```sh
-> show outbox at level 0
-Outbox has N messages:
-{ unparsed_parameters: ..;
-  destination: ..;
-  entrypoint: ..; }
-..
-```
+During the refutation period for a commitment period, if two or more nodes publish different commitments, two of them play a _refutation game_ to identify the correct commitment.
+The nodes automatically play the refutation game by stepping through their logic to identify the point at which they differ.
+At this point, layer 1 uses the PVM to identify the correct commitment.
+Then the protocol gives half of the incorrect commitment's stake to the correct commitment's stake, and burns the other half.
+Then the protocol eliminates the incorrect commitment because it has no stake.
 
-The reveal channel described previously is available in the debugger,
-either automatically or through specific commands. The debugger can automatically fill preimages from files in a specific directory on the disk, by default in the `preimage` subdirectory of the working directory. It can be configured with the option `--preimage-dir <directory>`.
+This refutation game happens as many times as is necessary to eliminate incorrect commitments.
+Because the node that ran the PVM correctly is guaranteed to win the refutation game, a single honest node is enough to ensure that the Smart Rollup is running correctly.
+For this reason, Smart Rollups are sometimes called Smart Optimistic Rollups.
```

Comment:
"This kind of Smart Rollup is called a Smart Optimistic Rollup because the commitments are assumed to be correct until they are proven wrong by an honest rollup node."